plugins/obs-ffmpeg/obs-ffmpeg-source.c

```diff
@@ -94,6 +111,11 @@ static const char *video_filter =
 static const char *audio_filter =
 	" (*.mp3 *.aac *.ogg *.wav);;";
 
+static char* get_latest_replay_buffer()
```

Comment:
The asterisk should immediately precede the function name.
"char* get_latest_replay_buffer" should be "char *get_latest_replay_buffer"